File: src/client/xnative/lion.xdroid/Control/MainPagerAdapter.cs
using System.Collections.Generic;
using Android.Support.V4.App;
using Java.Lang;

namespace Lion.XDroid.Control
{
    public class MainPagerAdapter : FragmentPagerAdapter
    {
        private List<Fragment> fragment_list = new List<Fragment>();
        private List<string> title_list = new List<string>();
        private List<bool> anonymous_list = new List<bool>();

        public MainPagerAdapter(FragmentManager fm) 
                : base(fm)
        {
        }

        public void AddFragment(Fragment fragment, string title, bool is_anonymous)
        {
            this.fragment_list.Add(fragment);
            this.title_list.Add(title);
            this.anonymous_list.Add(is_anonymous);
        }

        public bool IsAnonymous(int position)
        {
            return anonymous_list[position];
        }

        public override Fragment GetItem(int position)
        {
            return this.fragment_list[position];
        }

        public override int Count
        {
            get
            {
                return this.fragment_list.Count;
            }
        }

        public override ICharSequence GetPageTitleFormatted(int position)
        {
            return new Java.Lang.String(title_list[position]);
        }
    }
}

File: src/client/xcode/Lion.Droid/app/src/main/java/com/odinsoftware/lottolion/control/MainPagerAdapter.java
package com.odinsoftware.lottolion.control;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;
import android.support.v4.view.PagerAdapter;

import java.util.ArrayList;

/**
 * Created by limyg on 2017-04-24.
 */
public class MainPagerAdapter extends FragmentPagerAdapter {
    public interface MainPager{
        int 추첨결과 = 0;
        int 번호관리 = 1;
        int 로또판매점 = 2;
        int 설정 = 3;
    }
    private final ArrayList<Fragment> mFragments = new ArrayList<>();
    public MainPagerAdapter(FragmentManager fm) {
        super(fm);
    }
    public void addFragment(Fragment fragment)
    {
        mFragments.add(fragment);
    }
    @Override
    public Fragment getItem(int position) {
        return mFragments.get(position);
    }

    @Override
    public int getCount() {
        return mFragments.size();
    }

    @Override
    public CharSequence getPageTitle(int position) {
        switch (position) {
            case MainPager.추첨결과:
                return "추첨결과";
            case MainPager.번호관리:
                return "번호관리";
            case MainPager.로또판매점:
                return "로또판매점";
            case MainPager.설정:
                return "설정";
        }
        return "알수없음";
    }

}
